samples/snippets/src/main/java/com/example/monitoring/UpdateAlertPolicy.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.monitoring;
+
+// [START monitoring_update_alert_policy]
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.monitoring.v3.AlertPolicyServiceClient;
+import com.google.monitoring.v3.Aggregation;
+import com.google.monitoring.v3.AlertPolicy;
+import com.google.monitoring.v3.ComparisonType;
+import com.google.monitoring.v3.UpdateAlertPolicyRequest;
+import com.google.protobuf.Duration;
+import java.io.IOException;
+
+// Sample to update an alert policy
+public class UpdateAlertPolicy {
+
+  public static void main(String[] args) throws ApiException, IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String alertPolicyName = "your-alert-policy-displayname";
+    String updatedAlertPolicyName = "your-updated-alert-policy-displayname";
+    updateAlertPolicy(alertPolicyName, updatedAlertPolicyName);
+  }
+
+  public static void updateAlertPolicy(String alertPolicyName, String newPolicyName)
+      throws ApiException, IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      // A Filter that identifies which time series should be compared with the threshold
+      String metricFilter =
+          String.format(
+              "metric.type=\"compute.googleapis.com/instance/"
+                  + "cpu/utilization\" AND resource.type=\"gce_instance\"");
```

Comment:
Same issue, can we make it easier to read and to edit for customers?

Trying a different approach this time.
```java
String metricFilter = 
        "metric.type="
        + '"' + compute.googleapis.com/instance/cpu/utilization" + '"'
        + " AND "
        + "resource.type="
        + '"' + "gce_instance" + '"";
```

Reply:
Done